tests/Functional/Net/SFTPUserStoryTest.php

```diff
@@ -649,5 +649,22 @@ public function testStatVsLstat($sftp)
         $this->assertSame($stat['type'], NET_SFTP_TYPE_SYMLINK);
 
         $sftp->enableStatCache();
+
+        return $sftp;
+    }
+
+    /**
+     * @depends testStatVsLstat
+     * @group github830
+     */
+    public function testEndlessLoopOnUpload($sftp)
```

Comment:
@terrafrost Are you saying that #830 was not acceptable after all? If yes, why are you including the second hunk?